Neos.FluidAdaptor/Configuration/Settings.yaml

```diff
@@ -56,3 +56,9 @@ Neos:
           options:
             namespaces:
               f: 'Neos\FluidAdaptor\ViewHelpers'
+
```

Comment:
Have you given any thought to move the content of the `getDefaultNamespaces` method into this syntax?

Reply:
Yes, in my opinion getDefaultNamespaces() could then be fully replaced by setting the f namespaces via configuration. I tried this and removed the constructor and the getDefaultNamespaces method in the ViewHelperResolver class and it worked for me.
I also tried to simply inject the configuration into the existing $namespaces variable, which also worked. This way the new variable $defaultNamespacesFromConfiguration and the loop for adding them wouldn't be needed . But I'm not sure if this would be a good practice? Also the sanity checks performed by addNamespace(), wouldn't be executed this way.

Reply:
It's important that we keep the sanity check, so let's inject the configuration to a property and loop that 👍 

And replace `getDefaultNamepsaces` and move the configuration to Settings.yaml. I would love to hear from @albe or @bwaidelich if they see anything against doing that?